### PR TITLE
Feature/check canopy

### PR DESCRIPTION
**Pull request by LarissaReames-NOAA**
Addresses Issue #481

### CODE REVIEW

File: sorc/chgres_cube.fd/input_data.F90 [hunk 1]
```diff
@@ -6639,4 +6640,27 @@ subroutine check_soilt(soilt, landmask, skint)
     enddo
   enddo
 end subroutine check_soilt
+
```
Comment (GeorgeGayno-NOAA) [inline sorc/chgres_cube.fd/input_data.F90:6643]: As an FYI, the 0.5 is the maximum value used by the Noah LSM. And it is in millimeters. I would add the units to the documentation.

**Comment by GeorgeGayno-NOAA** [comment]
Do the consistency tests pass?

**Comment by GeorgeGayno-NOAA** [comment]
The consistency checks passed on Orion, Jet, Hera, WCOSS-Cray and WCOSS-Dell.

File: sorc/chgres_cube.fd/input_data.F90 [hunk 2]
```diff
@@ -48,7 +48,7 @@ module input_data
 
  private
 
- public :: check_soilt
+ public :: check_soilt, check_cnwat
```
Comment (edwardhartnett) [inline sorc/chgres_cube.fd/input_data.F90:51]: These need doxygen documentation.

Comment (LarissaReames-NOAA) [reply sorc/chgres_cube.fd/input_data.F90:51]: I'm not sure what you're asking for here. The subroutines themselves have documentation at the top of each one. Do you just want me to write out to the side that these are subroutines contained in the module?

Comment (edwardhartnett) [reply sorc/chgres_cube.fd/input_data.F90:51]: Sorry I thought they were module variables...

**Comment by GeorgeGayno-NOAA** [comment]
@LarissaReames-NOAA Let's merge this next. Please update your branch.

**Comment by LarissaReames-NOAA** [comment]
Branch has been updated with develop and passes all tests. It's ready to be merged.

**Review by GeorgeGayno-NOAA — APPROVED** [review]
Looks good.

**Pull request merged**